data/sql/updates/pending_db_world/rev_1633365891402740400.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1633365891402740400');
+
+UPDATE `quest_template` SET `AllowableRaces`=690 WHERE `ID`=5405;
+UPDATE `quest_template` SET `AllowableRaces`=1101 WHERE `ID`=5401;
+UPDATE `creature_queststarter` SET `id`=11039 WHERE `quest`=5503;
+UPDATE `creature_questender` SET `id`=11039 WHERE `quest`=5503;
```

Comment:
```suggestion
UPDATE `creature_questender` SET `id`=11039 WHERE `quest`=5405;
```
See https://tbc.wowhead.com/quest=5405/argent-dawn-commission